piece_of_cake.py

```diff
@@ -0,0 +1,7 @@
+def get_recipe_price(prices, optionals=[], **ingredients):
```

Comment:
Please add documantation to your function, and try to simplify your code as mentioned in the other notes i gave you :)